src/main/rx/flysky.c

```diff
@@ -442,6 +452,21 @@ rx_spi_received_e flySkyDataReceived (uint8_t *payload)
 
     if (bound) {
         checkTimeout();
+#ifdef USE_RX_FLYSKY_SPI_LED
+        if (result == RX_SPI_RECEIVED_DATA) {
+            led_rxlosscount = 0;
+            IOHi(flySkyLedPin);
+        } else {
+            led_rxlosscount++;
+            if (led_rxlosscount > 1000) {
+                led_rxlosscount = 1001;
+                if ((micros() % 2000000) > 1000000)
```

Comment:
This is a bit awkward too. Probably better to use milliseconds, and use a static variable to store the last led change:

```
static timeMs_t ledLastUpdate = 0;
static bool ledOn = false;
timeMs now = millis();
if (now - ledLastUpdate > INTERVAL_RX_LOSS_MS) {
    ledLastUpdate = now;
    if (ledOn) {
        IOLo(flySkyLedPin);
    } else {
        IOHi(flySkyLedPin);
    }
    ledOn = !ledOn;
}
````

Also, use defines with explanatory names for constant numbers.